sw/device/lib/testing/lc_ctrl_testutils.h

```diff
@@ -0,0 +1,36 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_TESTING_LC_CTRL_TESTUTILS_H_
+#define OPENTITAN_SW_DEVICE_LIB_TESTING_LC_CTRL_TESTUTILS_H_
+
+#include <stdbool.h>
+
+#include "sw/device/lib/dif/dif_lc_ctrl.h"
+
+/**
+ * Lifecycle Controller handle.
+ */
+typedef struct lc_ctrl {
+  /**
+   * Lifecycle Controller peripheral interface.
+   */
+  dif_lc_ctrl_t dif;
+} lc_ctrl_t;
```

Comment:
Likewise, this doesn't provide anything over the existing DIF handle. Can remove and use the DIF handle directly in a test.

Reply:
Done.